express/features/support/steps/client-steps.ts

```diff
@@ -30,3 +31,9 @@ Given("the user submits a valid public key without headers", async function () {
     await enterTextIntoTextInput(this.page, VALID_PUBLIC_KEY_WITHOUT_HEADERS, "serviceUserPublicKey");
     await clickSubmitButton(this.page);
 });
+
+Then("they should see the public key they just entered in the textarea", async function () {
+    const textArea = await this.page.$("#serviceUserPublicKey");
+    const textAreaContent = await this.page.evaluate((element: {textContent: string}) => element.textContent, textArea);
```

Comment:
```suggestion
Then("they should see the public key they just entered in the textarea", async function (this: TestContext) {
    const textArea = await this.page.$("#serviceUserPublicKey");
    const textAreaContent = await this.page.evaluate(element => element.textContent, textArea);
```

Tell the interpreter the type of `this` and then it knows the type of `element` and we don't need the explicit type declaration (`element` is of type `any` but it's not implicit since it's specified by the `Page` type).